FBSnapshotTestCase/FBSnapshotTestCasePlatform.m

```diff
@@ -34,7 +34,7 @@ BOOL FBSnapshotTestCaseIs64Bit(void)
 NSString *FBDeviceAgnosticNormalizedFileName(NSString *fileName)
 {
   UIDevice *device = [UIDevice currentDevice];
-  CGSize screenSize = [[UIApplication sharedApplication] keyWindow].bounds.size;
```

Comment:
That's true, but I figured using the screen would:

- Be equivalent to the window in most cases
- Work when the library is integrated as a framework (where `keyWindow` does not)

Maybe a more comprehensive way to solve this would be to generate the suffix based on the view being tested, which is probably what you really want anyway.  This makes even more sense when you consider that snapshots are generated based on the bounds of the specified view, even creating and "keying" a window if necessary.[0]

0: https://github.com/facebook/ios-snapshot-test-case/blob/master/FBSnapshotTestCase/Categories/UIImage+Snapshot.m#L46